meta-lmp-bsp/recipes-bsp/u-boot/u-boot-fio_%.bbappend

```diff
@@ -1,4 +1,12 @@
-FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
+FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:${THISDIR}/files:"
+
+SRC_URI_append_imx = " \
+    file://0001-optee-obtain-emmc-rpmb-info-from-dt.patch \
+"
+
+SRC_URI_append_apalis-imx6-sec = " \
+    file://0001-arch-arm-dts-imx6-apalis-u-boot-Add-RPMB-dev-ID.patch \
```

Comment:
We should be safe to have both patches as part of our u-boot-fio tree / branch I would think. We maintain our own fork, and the 2020.04 is basically for imx targets anyway.

Reply:
https://github.com/foundriesio/u-boot/pull/71